ht/cogs/roll.py

```diff
@@ -45,6 +49,60 @@ async def armiger(self, ctx, user: converters.Armiger = None):
 		await self.add_rolls(embed, "AND NOT personal", user, "Artist gallery")
 		await ctx.send(embed = embed)
 
+	@commands.command(
```

Comment:
```suggestion
        BOLD = re.compile("<i>|</i>")
	ITALIC = re.compile("<b>|</b>")
	OTHER_TAG = re.compile(r"<[^>]*>")

	@commands.command(
```

When using regexes in this codebase, I always use `re.compile`, as that way the regex only has to be parsed once by Python, and it is more performant. The `discord.py` library which this bot is based off does this too.